MIGRATING.md

```diff
@@ -64,32 +79,39 @@ OpenTelemetry Java Instrumentation.
 ### Configuration setting changes
 
 These SignalFx Java Agent system properties correspond to the following
-OpenTelemetry system properties:
+OpenTelemetry system properties (NOTE: some properites are exporter - specific, the default is `jaeger`):
 
 | SignalFx system property | OpenTelemetry system property |
 | ------------------------ | ----------------------------- |
-| `signalfx.service.name` | `otel.resource.attributes=service.name=my-java-app` |
+| `signalfx.service.name` | `otel.resource.attributes=service.name=<name of the service>` |
+| `signalfx.env` | `otel.resource.attributes=deployment.environment=<name of the environment>` |
 | `signalfx.endpoint.url` | `otel.exporter.jaeger.endpoint` |
-| `signalfx.tracing.enabled` | `otel.trace.enabled` |
-| `signalfx.span.tags` | `otel.resource.attributes` |
-| `signalfx.recorded.value.max.length` | `otel.config.max.attr.length` |
-| `signalfx.db.statement.max.length` | `otel.config.max.attr.length` | 
+| `signalfx.tracing.enabled` | `otel.javaagent.enabled` |
+| `signalfx.integration.<name>.enabled=false` | `otel.instrumentation.[id].enabled=false` | 
```

Comment:
why the different formatting for placeholders here?

Reply:
Copied from dependencies (signalfx / otel). Can make it uniform tho.